package/Shaders/Utility.hlsl

```diff
@@ -363,1037 +364,52 @@ cbuffer AlphaTestRefCB : register(b11)
-		{ 0.363842, 0.965911 },
-		{ 0.711875, 0.909482 },
-		{ 0.546220, 0.903623 },
-		{ 0.772271, 0.548479 },
-		{ 0.068636, 0.239692 },
-		{ 0.510727, 0.662648 },
-		{ 0.822840, 0.200873 },
-		{ 0.847102, 0.728782 },
-		{ 0.803949, 0.309091 },
-		{ 0.165929, 0.999207 },
-		{ 0.017029, 0.258339 },
-		{ 0.144047, 0.691153 },
-		{ 0.306345, 0.882595 },
-		{ 0.285745, 0.823756 },
-		{ 0.900418, 0.215857 },
-		{ 0.790826, 0.652181 },
-		{ 0.542619, 0.093509 },
-		{ 0.987152, 0.404187 },
-		{ 0.586779, 0.546159 },
-		{ 0.001221, 0.148381 },
-		{ 0.388684, 0.041261 },
-		{ 0.303323, 0.654195 },
-		{ 0.229987, 0.413099 },
-		{ 0.089267, 0.055513 },
-		{ 0.050783, 0.441877 },
-		{ 0.165960, 0.925779 },
-		{ 0.369152, 0.439344 },
-		{ 0.994201, 0.842128 },
-		{ 0.601642, 0.237831 },
-		{ 0.259865, 0.319346 },
-		{ 0.311930, 0.094302 },
-		{ 0.850368, 0.584216 },
-		{ 0.444411, 0.943205 },
-		{ 0.841090, 0.613819 },
-		{ 0.752647, 0.868892 },
-		{ 0.842311, 0.476882 },
-		{ 0.998779, 0.556658 },
-		{ 0.728843, 0.299997 },
-		{ 0.419080, 0.511399 },
-		{ 0.006531, 0.190100 },
-		{ 0.777184, 0.865688 },
-		{ 0.552416, 0.808313 },
-		{ 0.915708, 0.267495 },
-		{ 0.238685, 0.957274 },
-		{ 0.177679, 0.673330 },
-		{ 0.420118, 0.755242 },
-		{ 0.690115, 0.375103 },
-		{ 0.646199, 0.485824 },
-		{ 0.147954, 0.874203 },
-		{ 0.453780, 0.558763 },
-		{ 0.876095, 0.776513 },
-		{ 0.097812, 0.453566 },
-		{ 0.288797, 0.782952 },
-		{ 0.518113, 0.927244 },
-		{ 0.534715, 0.804621 },
-		{ 0.611652, 0.132572 },
-		{ 0.395886, 0.367779 },
-		{ 0.074587, 0.781732 },
-		{ 0.097903, 0.731346 },
-		{ 0.478469, 0.659139 },
-		{ 0.746086, 0.178991 },
-		{ 0.882809, 0.659932 },
-		{ 0.502335, 0.552293 },
-		{ 0.618427, 0.745140 },
-		{ 0.033296, 0.215278 },
-		{ 0.820429, 0.939238 },
-		{ 0.218726, 0.918638 },
-		{ 0.965300, 0.902982 },
-		{ 0.412519, 0.327219 },
-		{ 0.575793, 0.116550 },
-		{ 0.498398, 0.813318 },
-		{ 0.800562, 0.804651 },
-		{ 0.014649, 0.716178 },
-		{ 0.281167, 0.252632 },
-		{ 0.460189, 0.991974 },
-		{ 0.858425, 0.487960 },
-		{ 0.405591, 0.226936 },
-		{ 0.619953, 0.344584 },
-		{ 0.225959, 0.730461 },
-		{ 0.876553, 0.384533 },
-		{ 0.740471, 0.556719 },
-		{ 0.335887, 0.937803 },
-		{ 0.644612, 0.585192 },
-		{ 0.090213, 0.833979 },
-		{ 0.772027, 0.838069 },
-		{ 0.631825, 0.548631 },
-		{ 0.592944, 0.926115 },
-		{ 0.686911, 0.899899 },
-		{ 0.655538, 0.811243 },
-		{ 0.508652, 0.126316 },
-		{ 0.732109, 0.202979 },
-		{ 0.587268, 0.010498 },
-		{ 0.123875, 0.837397 },
-		{ 0.554979, 0.207617 },
-		{ 0.063143, 0.175512 },
-		{ 0.594287, 0.775018 },
-		{ 0.201819, 0.809259 },
-		{ 0.236732, 0.134098 },
-		{ 0.987640, 0.088839 },
-		{ 0.818842, 0.427076 },
-		{ 0.110019, 0.473128 },
-		{ 0.725730, 0.838618 },
-		{ 0.752953, 0.201086 },
-		{ 0.549883, 0.498978 },
-		{ 0.399152, 0.783258 },
-		{ 0.449843, 0.307382 },
-		{ 0.303385, 0.014557 },
-		{ 0.442976, 0.371654 },
-		{ 0.449446, 0.846675 }
+		float2(0.105594f, 0.25663f) * 2.0 - 1.0,
```

Comment:
You are using that app again? If it is 8 there are existing charts, like in my blur shader. Probably should put it somewhere common tho.

Reply:
probably should be shared yeah